src/plugins/unified_doc_viewer/public/components/doc_viewer_logs_overview/sub_components/highlight_field.tsx

```diff
@@ -39,12 +38,19 @@ export function HighlightField({
   children,
   ...props
 }: HighlightFieldProps) {
+  const { euiTheme } = useEuiTheme();
+
   const hasFieldDescription = !!fieldMetadata?.short;
 
   return formattedValue && value ? (
     <div {...props}>
       <EuiFlexGroup responsive={false} alignItems="center" gutterSize="xs">
-        <EuiTitle css={fieldNameStyle} size="xxxs">
+        <EuiTitle
+          css={css`
+            color: ${euiTheme.colors.textHeading};
```

Comment:
Suggestion: This override is not necessary as `EuiTitle` already has the right color by default.

Reply:
Addressed with [style(logs): update style suggestions](https://github.com/elastic/kibana/pull/202746/commits/7b12855cc171332cadbb533ee98505855aa016b6)